src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java

```diff
@@ -265,11 +256,23 @@ public JSONObject getRepoPlanInfo(String accessToken, int repoId) {
     public JSONObject getTeamActivity(String accessToken) {
         try {
             JSONResponse jsonResponse = ClientUtils.sendGet(TEAM_ACTIVITY_ENDPOINT, accessToken);
-            jsonResponse.raiseForStatus();
             return jsonResponse.getJsonResponse();
         } catch (RequestError | InvalidJsonError | StatusCodeError error) {
             CodeSyncLogger.error(String.format("Error while getting team activity data. %s", error.getMessage()));
             return null;
         }
     }
+
+    public JSONObject reactivateAccount(String accessToken) {
+        try {
+            JSONResponse jsonResponse = ClientUtils.sendPost(API_REACTIVATE_ACCOUNT, accessToken);
```

Comment:
I think we discussed this, we need to use Browser/Webapp for this instead of API
![image](https://github.com/codesyncapp/codesync-intellij/assets/9863016/f953ffb2-9af7-4df4-b7ae-4a1156746d48)

Slack: https://codesyncapp.slack.com/archives/C015NSBKNQH/p1704608068717999?thread_ts=1704569824.082849&cid=C015NSBKNQH

Reply:
Oh yes! forgot to remove this. I added this before our discussion. It is not being used anywhere. I will remove this now.